addons/website_event_exhibitor/controllers/website_event_main.py

```diff
@@ -12,6 +12,7 @@ class WebsiteEventController(main.WebsiteEventController):
     def _prepare_event_register_values(self, event, **post):
         values = super(WebsiteEventController, self)._prepare_event_register_values(event, **post)
 
+        # TODO remove in master, unused
```

Comment:
I was debating that at first but I think it will crash if people extended that controller somewhere, which is possible.

So for safety, would keep a "blank" method that just returns super.

Reply:
@awa-odoo fair enough, added back the file with just a call to super